pkg/policies/flowcontrol/selectors/control-point-id.go

```diff
@@ -10,19 +10,36 @@ import (
 // ControlPointID is the struct that represents a ControlPoint.
 //
 // Agent group is implied.
-//
-// Note: We need to mirror cmdv1.ServiceControlPoint, because
-// protobuf-generated struct cannot be used as map keys.
+// Type is ignored.
 type ControlPointID struct {
 	Service      string
 	ControlPoint string
 }
 
+// TypedControlPointID is the struct that represents a FlowControlPoint.
+//
+// Agent group is implied.
+//
+// Note: We need to mirror flowcontrolpointsv1.FlowControlPoint, because
+// protobuf-generated struct cannot be used as map keys.
+type TypedControlPointID struct {
+	ControlPointID
+	Type string
+}
+
 // GlobalControlPointID is ControlPointID with explicit agent group.
 //
-// Note: We need to mirror cmdv1.ServiceControlPoint, because
+// Note: We need to mirror cmdv1.GlobalFlowControlPoint, because
 // protobuf-generated struct cannot be used as map keys.
 type GlobalControlPointID struct {
+	TypedControlPointID
+	AgentGroup string
+}
+
+// UntypedGlobalControlPointID is just like GlobalControlPointID but embedding the ControlPointID instead of TypedControlPointID.
+//
+// Useful for defining a control point to find, without having to specify the source.
+type UntypedGlobalControlPointID struct {
```

Comment:
Nit: This assymetry in naming is a bit weird (global is "by default" Typed and local is by default "Untyped").

Reply:
I know. That untyped global was added in the end. But I think it's worth refactoring to be consistent.

Reply:
Updated.